### ISSUE TITLE
Please add support for iOS 14.x

### ISSUE THREAD

**Issue by GTO683** [Galactic-Dev/NotFinical#3]
NotFinical isn't working for me on iOS 14. The gestures just don't dismiss anything, nor does the timer work.

**Comment by Galactic-Dev** [comment]
what version are you using 1.0.2 added ios 14 support

**Comment by GTO683** [comment]
1.0.2, 14.3, XR

**Comment by Galactic-Dev** [comment]
The tweak works on my 14.3 XR, could you give a tweak list

**Comment by GTO683** [comment]
Ok so I figured out 2 things. 1.) I didn't see at the bottom of the settings page that you wrote that the media player has to be paused in order for the timer feature to work. After realizing that I tested it and it does disappear after the duration I set. that works.

However, the swiping to dismiss one doesn't work. To make sure, I disabled every tweak in iCleanerPro besides NotFinical and tried the swiping and it still didn't work.

So in conclusion, the timer to dismiss works, but the swipe to dismiss doesn't. For me atleast.
